src/GoTrueClient.ts

```diff
@@ -660,10 +660,6 @@ export default class GoTrueClient {
     }
   }
 
-  /**
-   * Returns the session, refreshing it if necessary.
```

Comment:
Ah is this section no longer needed/are we making `getSession` internal? Not sure if removing the comments will remove the description from reference spec

Reply:
Hmm I'll investigate.